templates/layouts/main_layout.html

```diff
@@ -21,19 +23,25 @@
 alt="SenNet logo" /></a>
 
 <div class="collapse navbar-collapse">
-<ul class="navbar-nav">
-<li class="nav-item">
-<a class="nav-link" href="https://sennetconsortium.org/" target="_blank">Member Portal</a>
-</li>
-{% if session['isAdmin'] %}
-<li class="nav-item">
-<a class="nav-link" href="{{ config['FLASK_APP_BASE_URI'] }}/registrations">Registrations</a>
-</li>
-<li class="nav-item">
-<a class="nav-link" href="{{ config['FLASK_APP_BASE_URI'] }}/members">Members</a>
-</li>
-{% endif %}
-</ul>
+    <ul class="navbar-nav">
+        <li class="nav-item">
+            <a class="nav-link" href="https://sennetconsortium.org/" target="_blank">Member Portal</a>
+        </li>
+        {% if session['isAdmin'] %}
+        <li class="nav-item">
+            <a class="nav-link" href="{{ config['FLASK_APP_BASE_URI'] }}/registrations">Registrations</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link" href="{{ config['FLASK_APP_BASE_URI'] }}/members">Members</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link state-element" id="js-btn--downloads-members" href="#" data-js-apicall="{{ config['FLASK_APP_BASE_URI'] }}/downloads/members" data-js-toastit="Your download will begin shortly ..." data-filename="SenNet-members-list.csv">
+                Download Member Info
```

Comment:
Can we move this download link to the Members page? 

![image](https://user-images.githubusercontent.com/195873/200908932-0558251c-5ff7-4f75-a365-336e2ce54b6a.png)

Reply:
@yuanzhou  Sure, I put the link there because this the original request from the the issue #4 card. Can you confirm you still want it moved?

Reply:
Bill wanted to keep it on the Nav menu.